janusgraph-core/src/main/java/org/janusgraph/graphdb/transaction/StandardJanusGraphTx.java

```diff
@@ -1576,4 +1576,11 @@ public boolean hasModifications() {
         return !addedRelations.isEmpty() || !deletedRelations.isEmpty();
     }
 
+    @Override
+    public void expireSchemaElement(final long id) {
+        final InternalVertex v = vertexCache.get(id, externalVertexRetriever);
```

Comment:
This call will fetch this vertex from the storage backend if it does not exist in the cache, which is unnecessary. I think a better way is to create a new method, e.g. `getIfPresent` in `GuavaVertexCache.java`.

Reply:
Thanks, I used the existing method `contains`.